packages/ckeditor5-dev-docs/lib/web-crawler/index.js

```diff
@@ -76,9 +77,8 @@ module.exports = async function verify( options ) {
 
 	logErrors( errors );
 
-	if ( errors.size ) {
-		process.exit( 1 );
-	}
+	// Always exit the script because `spinner` can freeze the process of the crawler is executed in the `noSpinner:true` mode.
```

Comment:
```suggestion
	// Always exit the script because `spinner` can freeze the process of the crawler if it is executed in the `noSpinner:true` mode.
```